verifyFiles.py

```diff
@@ -193,13 +198,38 @@ def pretty_string(diff_dict,conditions_dict):
         output_string+="\n"
     return output_string
 
+#Method is_subject_folders_same checks if the subject_folders under different conditions are the same. If not , it stops the execution of the script.
+def is_subject_folders_same(conditions_list,root_dir):
+   if len(conditions_list) > 1:
+      subject_folders_ref_list=[subject_name for subject_name in sorted(os.listdir(os.path.join(root_dir,conditions_list[0]))) if os.path.isdir(os.path.join(root_dir,conditions_list[0],subject_name))]
```

Comment:
Okay. I read it from file system because, this is happening even before the get_file_dict operation right ? 
In that case, we should do this check after the reading of dictionaries. Is that the better way ?